src/createBoundingBox.js

```diff
@@ -0,0 +1,65 @@
+//Update popper position
+module.exports.updatePopperObjectPosition = function (target) {
+    let popper = target.scratch('popper');
+    popper.scheduleUpdate();
+    return popper;
+};
+
+//Wrap given bounding Box to match popper.js bounding box
+module.exports.getPopperBoundingBox = function (target, userOptions) {
+
+    let {position, cy, getDimensions} = userOptions;
+    let cyOffset = cy.container().getBoundingClientRect();
+    let dimensions = getDimensions(target);
+    let renderedPosition = position(target);
+    let scrollY = window.pageYOffset;
+    let scrollX = window.pageXOffset;
+   
+    //Exit if position is invalid
```

Comment:
For what cases would the position be invalid?  Is it possible to set the defaults for `options` so that this doesn't happen by default?  If not, maybe we should just throw an `Error` since returning `undefined` would just cause an error farther away from the cause.

Reply:
Since position() can be a user defined function, its possible that the function could return an error as the user could technically pass a function that doesn't properly work with all targets. So, I thought It would be better to catch the error instead of letting something break further down. 

I have modified to the code to throw an error with a descriptive mesage